Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM jinaai/jina:2-py37-perf
+
+# install requirements before copying the workspace
+COPY requirements.txt /requirements.txt
```

Comment:
This Dockerfile here is the standard Dockerfile that is added automatically by hubble. I don't think we need it here @nan-wang right?

Reply:
jakob is correct. Let's wait the response from the hubbler. https://jinaai.slack.com/archives/C023A7RAPGR/p1634715971023500